common/app/experiments/Experiments.scala

```diff
@@ -25,7 +25,7 @@ object LiveblogRendering
       participationGroup = Perc0A,
     )
 
-object PressedInteractives
+object ShowPressedInteractives
```

Comment:
This is probably for some historic reason, but just double checking that this should be classed as an experiment?

Reply:
Yeah, good question - it was originally added as an experiment and I've kept it as one. In my mind it's still more of an experiment than a switch. I retained the experiment in this PR to allow us to easily turn off the serving of pressed content if we start to notice unexpected errors and quickly want to revert to rendered content. After a period of time (say 2 weeks), in which we've built up confidence in the solution, I'd propose deleting the experiment altogether. 

What do you think, does it make sense?